src/libcore/num/flt2dec/mod.rs

```diff
@@ -410,8 +410,8 @@ fn determine_sign(sign: Sign, decoded: &FullDecoded, negative: bool) -> &'static
 /// it will only print given digits and nothing else.
 ///
 /// The byte buffer should be at least `MAX_SIG_DIGITS` bytes long.
-/// There should be at least 5 parts available, due to the worst case like
-/// `[+][0.][0000][45][0000]` with `frac_digits = 10`.
+/// There should be at least 4 parts available, due to the worst case like
+/// `[+][0.][0000][2][0000]` with `frac_digits = 10`.
```

Comment:
OK, perhaps the `+` should use a different bracket or no brackets, or just omit the `+` altogether. Also, why the `45` is changed to `2`? That doesn't seem to make any difference.